internal/version/version_test.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"testing"
+
+	"github.com/blang/semver"
+)
+
+func TestVersionSemver(t *testing.T) {
+	_, err := semver.Parse(version)
```

Comment:
Some context... while a version of Helm is being developed the version is set to the minor version only. You end up with versions output by the `version` command of `3.3+unreleased`.

When Helm is compiled `helm.sh/helm/v3/internal/version.version` is set as part of the processes with a semantic version. The `Makefile` does this for us.